docs_source_files/content/webdriver/keyboard.en.md

```diff
@@ -88,10 +88,10 @@ const {Builder, By, Key} = require('selenium-webdriver');
   {{< code-panel language="kotlin" >}}
 import org.openqa.selenium.By
 import org.openqa.selenium.Keys
-import org.openqa.selenium.chrome.ChromeDriver
+import org.openqa.selenium.firefox.FirefoxDriver

```

Comment:
Any reason to change it into Firefox from Chrome?

Reply:
Hi @manoj9788,

All other code samples in this section targets firefox browser. so made the change to make code sample matching to other

Reply:
Okay